Assets/scripts/NoteObject.cs

```diff
@@ -7,6 +7,7 @@ public class NoteObject : MonoBehaviour {
 	private Note _NoteData;
 	public float destroyY = -10f; // Y coordinate under which object will self destruct
 	public Vector3 velocity = new Vector3(0, -0.22f, 0);
+	public Vector3 centerOfObject = new Vector3(.5f, -.5f);

```

Comment:
OH, the prefab doesn't update when i update the script. Thats stupid. I'm fairly sure -0.5 is the correct value mathematically. Anything bigger than 0 means you can hit the note way after it passes the line it seems like.